### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [licyk/SDNote#1] [Comment]
# about/

https://sdnote.netlify.app/about/

**Comment by licyk** [comment]
<details>
<summary>配置</summary>

- CPU：R9 7940H
- 显卡：Radeon 780M / RTX 4060 Laptop 8G
- 内存：16 G
- 硬盘：1 TB + 512 GB

</details>


<details>
<summary>使用软件</summary>

SD WebUI：
- 仓库名：stable-diffusion-webui
- 远端地址：https://github.com/AUTOMATIC1111/stable-diffusion-webui
- 分支：dev
- 版本信息：ddb28b33a 2024-04-22 18:01:16

SD WebUI 扩展：
- 仓库名：a1111-sd-webui-haku-img
- 远端地址：https://github.com/KohakuBlueleaf/a1111-sd-webui-haku-img
- 分支：main
- 版本信息：5e8ea67 2024-03-11 13:51:30

- 仓库名：a1111-sd-webui-tagcomplete
- 远端地址：https://github.com/DominikDoom/a1111-sd-webui-tagcomplete
- 分支：main
- 版本信息：3eef536 2024-05-04 14:12:14

- 仓库名：adetailer
- 远端地址：https://github.com/Bing-su/adetailer
- 分支：main
- 版本信息：1edd588 2024-04-17 19:57:43

- 仓库名：advanced_euler_sampler_extension.git
- 远端地址：https://github.com/licyk/advanced_euler_sampler_extension.git
- 分支：main
- 版本信息：3bde72b 2024-05-10 20:55:15

- 仓库名：Euler-Smea-Dyn-Sampler
- 远端地址：https://github.com/licyk/Euler-Smea-Dyn-Sampler
- 分支：main
- 版本信息：4db83a2 2024-05-09 13:08:42

- 仓库名：LightDiffusionFlow
- 远端地址：https://github.com/Tencent/LightDiffusionFlow
- 分支：master
- 版本信息：1e9ac30 2023-12-24 21:52:10

- 仓库名：model-keyword
- 远端地址：https://github.com/mix1009/model-keyword
- 分支：main
- 版本信息：d08e874 2024-05-10 12:03:04

- 仓库名：multidiffusion-upscaler-for-automatic1111
- 远端地址：https://github.com/pkuliyi2015/multidiffusion-upscaler-for-automatic1111
- 分支：main
- 版本信息：574a096 2024-03-30 17:56:11

- 仓库名：PBRemTools
- 远端地址：https://github.com/mattyamonaca/PBRemTools
- 分支：main
- 版本信息：91e16d3 2023-08-01 19:08:11

- 仓库名：sd-dynamic-thresholding
- 远端地址：https://github.com/mcmonkeyprojects/sd-dynamic-thresholding
- 分支：master
- 版本信息：ec5d85c 2024-05-09 16:04:40

- 仓库名：sd-webui-animatediff
- 远端地址：https://github.com/continue-revolution/sd-webui-animatediff
- 分支：master
- 版本信息：44d2f99 2024-04-23 02:06:30

- 仓库名：sd-webui-boomer
- 远端地址：https://github.com/Haoming02/sd-webui-boomer
- 分支：main
- 版本信息：4f31031 2024-03-06 10:16:12

- 仓库名：sd-webui-cli-interruption
- 远端地址：https://github.com/light-and-ray/sd-webui-cli-interruption
- 分支：master
- 版本信息：72ebefa 2024-04-05 23:39:29

- 仓库名：sd-webui-controlnet
- 远端地址：https://github.com/Mikubill/sd-webui-controlnet
- 分支：main
- 版本信息：8d1389d 2024-05-07 17:58:48

- 仓库名：sd-webui-cutoff
- 远端地址：https://github.com/hnmr293/sd-webui-cutoff
- 分支：main
- 版本信息：33f102c 2023-11-14 01:02:22

- 仓库名：sd-webui-infinite-image-browsing
- 远端地址：https://github.com/zanllp/sd-webui-infinite-image-browsing
- 分支：main
- 版本信息：cadbe81 2024-04-20 23:15:51

- 仓库名：sd-webui-lora-block-weight
- 远端地址：https://github.com/hako-mikan/sd-webui-lora-block-weight
- 分支：main
- 版本信息：4d94d24 2024-04-06 16:32:18

- 仓库名：sd-webui-model-converter
- 远端地址：https://github.com/Akegarasu/sd-webui-model-converter
- 分支：main
- 版本信息：ae33f5c 2024-03-30 10:34:26

- 仓库名：sd-webui-neutral-prompt
- 远端地址：https://ghproxy.net/https://github.com/ljleb/sd-webui-neutral-prompt
- 分支：main
- 版本信息：64ee2be 2024-03-22 05:23:04

- 仓库名：sd-webui-openpose-editor
- 远端地址：https://github.com/huchenlei/sd-webui-openpose-editor
- 分支：main
- 版本信息：cebe13e 2023-12-18 22:58:26

- 仓库名：sd-webui-prompt-all-in-one
- 远端地址：https://github.com/Physton/sd-webui-prompt-all-in-one
- 分支：main
- 版本信息：eb924ba 2024-04-10 12:34:00

- 仓库名：sd-webui-regional-prompter
- 远端地址：https://github.com/hako-mikan/sd-webui-regional-prompter
- 分支：main
- 版本信息：50493ec 2024-04-06 23:53:11

- 仓库名：sd-webui-supermerger.git
- 远端地址：https://github.com/hako-mikan/sd-webui-supermerger.git
- 分支：main
- 版本信息：843ca28 2024-04-10 22:12:05

- 仓库名：sd-webui-tcd-sampler
- 远端地址：https://github.com/licyk/sd-webui-tcd-sampler
- 分支：main
- 版本信息：84fe4e0 2024-04-29 23:46:34

- 仓库名：stable-diffusion-webui-anti-burn
- 远端地址：https://github.com/klimaleksus/stable-diffusion-webui-anti-burn
- 分支：master
- 版本信息：6f4bf0a 2023-11-11 22:40:08

- 仓库名：stable-diffusion-webui-localization-zh_Hans
- 远端地址：https://github.com/hanamizuki-ai/stable-diffusion-webui-localization-zh_Hans
- 分支：master
- 版本信息：579a633 2024-04-23 15:00:03

- 仓库名：stable-diffusion-webui-model-toolkit
- 远端地址：https://github.com/arenasys/stable-diffusion-webui-model-toolkit
- 分支：master
- 版本信息：05b75f0 2024-03-25 08:19:36

- 仓库名：stable-diffusion-webui-rembg
- 远端地址：https://github.com/AUTOMATIC1111/stable-diffusion-webui-rembg
- 分支：master
- 版本信息：a4c07b8 2023-12-30 16:04:03

- 仓库名：stable-diffusion-webui-two-shot
- 远端地址：https://github.com/ashen-sensored/stable-diffusion-webui-two-shot
- 分支：main
- 版本信息：6b55dd5 2023-04-02 04:24:25

- 仓库名：sd-webui-wd14-tagger
- 远端地址：https://github.com/Akegarasu/sd-webui-wd14-tagger
- 分支：master
- 版本信息：3b57605 2024-03-09 15:59:56

- 仓库名：ultimate-upscale-for-automatic1111
- 远端地址：https://github.com/Coyote-A/ultimate-upscale-for-automatic1111
- 分支：master
- 版本信息：2322caa 2024-03-08 19:55:37

- 仓库名：z-a1111-sd-webui-dtg
- 远端地址：https://github.com/KohakuBlueleaf/z-a1111-sd-webui-dtg
- 分支：main
- 版本信息：979810e 2024-05-03 22:05:17

---

ComfyUI：
- 仓库名：ComfyUI
- 远端地址：https://github.com/comfyanonymous/ComfyUI
- 分支：master
- 版本信息：10fcd09 2024-04-27 00:48:41

ComfyUI 扩展：
- 仓库名：AIGODLIKE-COMFYUI-TRANSLATION
- 远端地址：https://github.com/AIGODLIKE/AIGODLIKE-COMFYUI-TRANSLATION
- 分支：main
- 版本信息：658e1c4 2024-04-26 17:37:23

- 仓库名：ComfyUI_Comfyroll_CustomNodes
- 远端地址：https://github.com/Suzie1/ComfyUI_Comfyroll_CustomNodes
- 分支：main
- 版本信息：d78b780 2024-01-24 22:44:09

- 仓库名：comfyui_controlnet_aux
- 远端地址：https://github.com/Fannovel16/comfyui_controlnet_aux
- 分支：main
- 版本信息：c0b3340 2024-04-05 20:07:00

- 仓库名：ComfyUI_Custom_Nodes_AlekPet
- 远端地址：https://github.com/AlekPet/ComfyUI_Custom_Nodes_AlekPet
- 分支：master
- 版本信息：093850f 2024-04-17 12:43:20

- 仓库名：ComfyUI_Cutoff
- 远端地址：https://github.com/BlenderNeko/ComfyUI_Cutoff
- 分支：master
- 版本信息：6c1b124 2024-04-09 00:34:21

- 仓库名：ComfyUI_experiments
- 远端地址：https://github.com/comfyanonymous/ComfyUI_experiments
- 分支：master
- 版本信息：934dba9 2023-09-13 02:28:20

- 仓库名：ComfyUI_IPAdapter_plus
- 远端地址：https://github.com/cubiq/ComfyUI_IPAdapter_plus
- 分支：main
- 版本信息：fef59c5 2024-04-23 09:17:56

- 仓库名：ComfyUI_restart_sampling
- 远端地址：https://github.com/ssitu/ComfyUI_restart_sampling
- 分支：main
- 版本信息：ea79890 2024-04-07 23:08:27

- 仓库名：ComfyUI_TiledKSampler
- 远端地址：https://github.com/BlenderNeko/ComfyUI_TiledKSampler
- 分支：master
- 版本信息：2fd9b05 2024-04-09 00:15:55

- 仓库名：ComfyUI_UltimateSDUpscale
- 远端地址：https://github.com/ssitu/ComfyUI_UltimateSDUpscale
- 分支：main
- 版本信息：b303386 2024-03-30 13:18:37

- 仓库名：ComfyUI-Advanced-ControlNet
- 远端地址：https://github.com/Kosinkadink/ComfyUI-Advanced-ControlNet
- 分支：main
- 版本信息：33d9884 2024-04-04 14:49:52

- 仓库名：ComfyUI-AnimateDiff-Evolved
- 远端地址：https://github.com/Kosinkadink/ComfyUI-AnimateDiff-Evolved
- 分支：main
- 版本信息：c3f61c6 2024-04-23 06:34:27

- 仓库名：comfyui-browser
- 远端地址：https://github.com/talesofai/comfyui-browser
- 分支：main
- 版本信息：fe649c9 2024-04-27 15:37:19

- 仓库名：ComfyUI-Crystools
- 远端地址：https://github.com/crystian/ComfyUI-Crystools
- 分支：main
- 版本信息：bfbfd12 2024-03-27 15:54:49

- 仓库名：ComfyUI-Custom-Scripts
- 远端地址：https://github.com/pythongosssss/ComfyUI-Custom-Scripts
- 分支：main
- 版本信息：3f2c021 2024-04-07 17:03:31

- 仓库名：ComfyUI-Extra-Samplers
- 远端地址：https://github.com/Clybius/ComfyUI-Extra-Samplers
- 分支：main
- 版本信息：52eac1b 2024-04-17 23:27:33

- 仓库名：ComfyUI-Impact-Pack
- 远端地址：https://github.com/ltdrdata/ComfyUI-Impact-Pack
- 分支：Main
- 版本信息：f7532d8 2024-04-27 16:39:07

- 仓库名：ComfyUI-Inspire-Pack
- 远端地址：https://github.com/ltdrdata/ComfyUI-Inspire-Pack
- 分支：main
- 版本信息：92cd510 2024-04-27 16:41:13

- 仓库名：ComfyUI-Manager
- 远端地址：https://github.com/ltdrdata/ComfyUI-Manager
- 分支：main
- 版本信息：ade3dee 2024-04-27 13:22:22

- 仓库名：ComfyUI-Marigold
- 远端地址：https://github.com/kijai/ComfyUI-Marigold
- 分支：main
- 版本信息：beb2c3b 2024-04-08 11:32:41

- 仓库名：ComfyUI-sonar
- 远端地址：https://github.com/blepping/ComfyUI-sonar
- 分支：main
- 版本信息：b78cbe0 2024-04-05 12:57:02

- 仓库名：ComfyUI-WD14-Tagger
- 远端地址：https://github.com/pythongosssss/ComfyUI-WD14-Tagger
- 分支：main
- 版本信息：d2d4828 2024-04-03 20:42:02

- 仓库名：comfyui-workspace-manager
- 远端地址：https://github.com/11cafe/comfyui-workspace-manager
- 分支：main
- 版本信息：acf12e5 2024-04-18 18:03:50

- 仓库名：ControlNet-LLLite-ComfyUI
- 远端地址：https://github.com/kohya-ss/ControlNet-LLLite-ComfyUI
- 分支：main
- 版本信息：103a445 2023-09-18 22:47:42

- 仓库名：Derfuu_ComfyUI_ModdedNodes
- 远端地址：https://github.com/Derfuu/Derfuu_ComfyUI_ModdedNodes
- 分支：master
- 版本信息：8aa0515 2024-04-27 10:27:10

- 仓库名：efficiency-nodes-comfyui
- 远端地址：https://github.com/jags111/efficiency-nodes-comfyui
- 分支：main
- 版本信息：3b7e89d 2024-04-06 00:43:29

- 仓库名：Euler-Smea-Dyn-Sampler
- 远端地址：https://github.com/licyk/Euler-Smea-Dyn-Sampler
- 分支：main
- 版本信息：0b80029 2024-04-17 15:03:27

- 仓库名：PowerNoiseSuite
- 远端地址：https://github.com/WASasquatch/PowerNoiseSuite
- 分支：main
- 版本信息：72b3f50 2023-09-19 10:03:53

- 仓库名：was-node-suite-comfyui
- 远端地址：https://github.com/WASasquatch/was-node-suite-comfyui
- 分支：main
- 版本信息：6c3fed7 2024-03-03 12:05:19

</details>


<details>
<summary>展开教程</summary>

![111](https://i.postimg.cc/RCDgG9Vz/Screenshot-2024-04-28-13-09-59-488-com-tencent-mobileqq.png)

</details>


<details>
<summary>经典案例</summary>

- 《星际玩家》

![pic](https://i.postimg.cc/JzQZGkS9/IMG-20240511-005140.jpg)


- 一起来猜猜

![pic](https://i.postimg.cc/tJYX0wz9/IMG-20240511-005105.jpg)


- 拍屏大师

![pic](https://i.postimg.cc/bwCfMDMz/IMG-20240511-005120.jpg)


- 有方法还看不懂

![pic](https://i.postimg.cc/gkwm1QYN/IMG-20240511-005049.jpg)


</details>

**Reply by licyk** [reply]
[《如何下载解压》](https://www.bilibili.com/video/BV18z42167dQ)

**Comment by KanoTheresaApocalypse** [comment]
li佬你好香啊~~

**Reply by licyk** [reply]
![???](https://i.postimg.cc/Y9FPPfLy/Screenshot-2024-04-30-17-05-21-844-com-tencent-mobileqq.png)

**Comment by nicocatxzc** [comment]
我喜欢你

**Comment by nicocatxzc** [comment]
很抱歉误操作pr了，如果对您造成了困扰，还请见谅
我在我们学校在导师的指导下需要做ai的普及讲座，
一些操作的结果演示直接引述您的文档案例比较方便，
感谢！

**Reply by licyk** [reply]
如果有需要，我可以将部分重构后内容合并进主分支并展示

**Reply by nicocatxzc** [reply]
按照您自己的进度来吧，没必要因我而改变您的进程。
相关资料我会自行整理的，
很感谢您能花费大量的精力进行操作、编撰、案例截取，编撰出如此完善的新手速通文字教程。
